2021-02-12 12:01:56,738 [Finding Names] [DEBUG] ['viper', '']
2021-02-12 12:01:59,877 [Finding Names] [DEBUG] ['viper', '']
2021-02-12 12:02:00,736 [Finding Names] [DEBUG] ['viper', '']
2021-02-12 12:23:06,285 [Finding Names] [DEBUG] ['viper', '']
2021-02-12 12:23:08,329 [Finding Names] [DEBUG] ['viper', '']
2021-02-12 12:23:10,218 [Finding Names] [DEBUG] ['viper', '']
2021-02-12 12:29:07,479 [Finding Names] [DEBUG] ['viper', '']
2021-02-13 09:05:07,926 [MainThread] [DEBUG] True
2021-02-13 09:05:23,500 [MainThread] [INFO] Script Runner has started
2021-02-13 09:05:23,611 [MainThread] [INFO] Assigning functions
2021-02-13 09:06:13,863 [Finding Names] [DEBUG] ['viper', '']
2021-02-13 09:44:51,691 [MainThread] [DEBUG] True
2021-02-13 09:44:51,737 [MainThread] [INFO] Configuration file has been loaded...
2021-02-13 09:44:51,738 [MainThread] [INFO] Script Runner has started
2021-02-13 09:44:51,787 [MainThread] [INFO] Assigning functions
2021-02-13 13:28:07,869 [MainThread] [DEBUG] True
2021-02-13 13:28:07,924 [MainThread] [INFO] Configuration file has been loaded...
2021-02-13 13:28:07,925 [MainThread] [INFO] Script Runner has started
2021-02-13 13:28:08,005 [MainThread] [INFO] Assigning functions
2021-02-13 13:28:15,028 [Finding Names] [DEBUG] ['viper', '']
2021-02-13 13:28:32,190 [Finding Names] [DEBUG] ['viper', '']
2021-02-13 13:28:35,107 [Finding Names] [DEBUG] ['viper', '']
2021-02-13 13:32:03,328 [MainThread] [DEBUG] True
2021-02-13 13:32:03,366 [MainThread] [INFO] Configuration file has been loaded...
2021-02-13 13:32:03,367 [MainThread] [INFO] Script Runner has started
2021-02-13 13:32:03,416 [MainThread] [INFO] Assigning functions
2021-02-13 13:32:24,960 [Finding Names] [DEBUG] ['viper', '']
2021-02-13 13:33:02,195 [Adding User] [INFO] This user will not be an admin
2021-02-13 13:33:02,207 [Adding User] [INFO] User testuser has been successfully added!
2021-02-13 13:37:40,748 [MainThread] [INFO] Assigning First Time Configurations Functions
2021-02-13 13:50:30,043 [MainThread] [DEBUG] True
2021-02-13 13:50:30,091 [MainThread] [INFO] Configuration file has been loaded...
2021-02-13 13:50:30,091 [MainThread] [INFO] Script Runner has started
2021-02-13 13:50:30,140 [MainThread] [INFO] Assigning functions
2021-02-13 15:23:23,924 [MainThread] [DEBUG] False
2021-02-13 15:23:23,979 [MainThread] [INFO] Configuration file has been loaded...
2021-02-13 15:23:23,980 [MainThread] [INFO] Script Runner has started
2021-02-13 15:23:24,230 [MainThread] [INFO] Assigning functions
2021-02-13 15:23:33,118 [Finding Names] [DEBUG] Elapsed time: 0.5386 seconds
2021-02-13 15:24:06,831 [Finding Names] [DEBUG] Elapsed time: 0.5220 seconds
2021-02-13 15:25:51,358 [MainThread] [DEBUG] False
2021-02-13 15:25:51,377 [MainThread] [INFO] Configuration file has been loaded...
2021-02-13 15:25:51,378 [MainThread] [INFO] Script Runner has started
2021-02-13 15:25:51,444 [MainThread] [INFO] Assigning functions
2021-02-13 15:25:56,626 [Finding Names] [DEBUG] Elapsed time: 0.5139 seconds
2021-02-13 15:28:18,752 [MainThread] [DEBUG] False
2021-02-13 15:28:18,771 [MainThread] [INFO] Configuration file has been loaded...
2021-02-13 15:28:18,771 [MainThread] [INFO] Script Runner has started
2021-02-13 15:28:18,839 [MainThread] [INFO] Assigning functions
2021-02-13 15:28:22,087 [MainThread] [DEBUG] Elapsed time: 0.5409 seconds
2021-02-13 15:28:27,081 [MainThread] [DEBUG] Elapsed time: 0.0000 seconds
2021-02-13 15:28:30,544 [MainThread] [DEBUG] Elapsed time: 0.0000 seconds
2021-02-13 15:31:22,646 [MainThread] [DEBUG] False
2021-02-13 15:31:22,665 [MainThread] [INFO] Configuration file has been loaded...
2021-02-13 15:31:22,665 [MainThread] [INFO] Script Runner has started
2021-02-13 15:31:22,732 [MainThread] [INFO] Assigning functions
2021-02-13 15:31:26,919 [MainThread] [DEBUG] Elapsed time: 0.5267 seconds
2021-02-13 15:31:29,378 [MainThread] [DEBUG] Elapsed time: 0.0000 seconds
2021-02-13 15:31:52,714 [MainThread] [DEBUG] Elapsed time: 0.0000 seconds
2021-02-13 15:31:58,442 [MainThread] [DEBUG] False
2021-02-13 15:31:58,462 [MainThread] [INFO] Configuration file has been loaded...
2021-02-13 15:31:58,462 [MainThread] [INFO] Script Runner has started
2021-02-13 15:31:58,529 [MainThread] [INFO] Assigning functions
2021-02-13 15:32:04,256 [MainThread] [DEBUG] Elapsed time: 3.0887 seconds
2021-02-13 15:32:15,198 [MainThread] [DEBUG] Elapsed time: 0.0000 seconds
2021-02-13 15:35:50,987 [MainThread] [DEBUG] False
2021-02-13 15:35:51,006 [MainThread] [INFO] Configuration file has been loaded...
2021-02-13 15:35:51,007 [MainThread] [INFO] Script Runner has started
2021-02-13 15:35:51,073 [MainThread] [INFO] Assigning functions
2021-02-13 15:37:50,849 [MainThread] [INFO] Output: ['Get-CimInstance : Access denied ', 'At C:\\WINDOWS\\system32\\WindowsPowerShell\\v1.0\\Modules\\BitLocker\\BitLocker.psm1:144 char:13', '+             Get-CimInstance `', '+             ~~~~~~~~~~~~~~~~~', '    + CategoryInfo          : PermissionDenied: (root\\cimv2\\Secu...cryptableVolume:String) [Get-CimInstance], CimException', '    + FullyQualifiedErrorId : HRESULT 0x80041003,Microsoft.Management.Infrastructure.CimCmdlets.GetCimInstanceCommand', ' ', 'Get-Win32EncryptableVolumeInternal :  does not have an associated BitLocker volume.', 'At C:\\WINDOWS\\system32\\WindowsPowerShell\\v1.0\\Modules\\BitLocker\\BitLocker.psm1:696 char:42', '+ ...       $AllWin32EncryptableVolume = Get-Win32EncryptableVolumeInternal', '+                                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~', '    + CategoryInfo          : NotSpecified: (:) [Write-Error], COMException', '    + FullyQualifiedErrorId : System.Runtime.InteropServices.COMException,Get-Win32EncryptableVolumeInternal', '']
2021-02-13 15:37:50,850 [MainThread] [INFO] [':', 'not', 'C:\\WINDOWS\\system32\\WindowsPowerShell\\v1.0\\Modules\\BitLocker\\BitLocker.psm1:696']
2021-02-13 15:38:49,619 [MainThread] [DEBUG] False
2021-02-13 15:38:49,638 [MainThread] [INFO] Configuration file has been loaded...
2021-02-13 15:38:49,638 [MainThread] [INFO] Script Runner has started
2021-02-13 15:38:49,706 [MainThread] [INFO] Assigning functions
2021-02-13 15:38:56,798 [MainThread] [INFO] Output: ['\r', '\r', '   ComputerName: MICHAEL-PC\r', '\r', 'VolumeType      Mount CapacityGB VolumeStatus           Encryption KeyProtector              AutoUnlock Protection\r', '                Point                                   Percentage                           Enabled    Status    \r', '----------      ----- ---------- ------------           ---------- ------------              ---------- ----------\r', 'Data            E:      3,725.90 FullyDecrypted         0          {}                                   Off       \r', 'Data            D:        930.97 FullyDecrypted         0          {}                                   Off       \r', 'OperatingSystem C:        930.43 FullyDecrypted         0          {}                                   Off       \r', 'Data            S:      1,863.00 FullyDecrypted         0          {}                                   Off       \r', 'Data            Q:         28.83 FullyDecrypted         0          {}                                   Off       \r', '\r', '\r', '']
2021-02-13 15:38:56,799 [MainThread] [INFO] ['E:', 'FullyDecrypted', 'D:', 'FullyDecrypted', 'C:', 'FullyDecrypted', 'S:', 'FullyDecrypted', 'Q:', 'FullyDecrypted']
2021-02-15 09:23:33,274 [MainThread] [DEBUG] True
2021-02-15 10:14:26,580 [Thread-25] [DEBUG] Starting new HTTP connection (48): 127.0.0.1:37013
2021-02-15 10:14:26,581 [MainThread] [ERROR] HTTPConnectionPool(host='127.0.0.1', port=37013): Max retries exceeded with url: /filter_and_sort_candidates (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0ef0707430>: Failed to establish a new connection: [Errno 111] Connection refused'))
2021-02-15 10:14:26,583 [Thread-8] [DEBUG] Starting new HTTP connection (49): 127.0.0.1:37013
2021-02-15 10:14:26,583 [MainThread] [ERROR] HTTPConnectionPool(host='127.0.0.1', port=37013): Max retries exceeded with url: /semantic_completion_available (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0e940fc610>: Failed to establish a new connection: [Errno 111] Connection refused'))
2021-02-15 10:14:26,611 [Thread-26] [DEBUG] Starting new HTTP connection (50): 127.0.0.1:37013
2021-02-15 10:14:27,756 [MainThread] [ERROR] HTTPConnectionPool(host='127.0.0.1', port=37013): Max retries exceeded with url: /semantic_completion_available (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0e940fc880>: Failed to establish a new connection: [Errno 111] Connection refused'))
2021-02-15 10:14:28,990 [Thread-14] [DEBUG] Starting new HTTP connection (51): 127.0.0.1:37013
2021-02-15 10:14:29,091 [MainThread] [ERROR] HTTPConnectionPool(host='127.0.0.1', port=37013): Max retries exceeded with url: /semantic_completion_available (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0e94130670>: Failed to establish a new connection: [Errno 111] Connection refused'))
2021-02-15 10:14:30,935 [Thread-27] [DEBUG] Starting new HTTP connection (52): 127.0.0.1:37013
2021-02-15 10:14:39,799 [Thread-5] [DEBUG] Starting new HTTP connection (53): 127.0.0.1:37013
2021-02-15 10:14:39,923 [MainThread] [ERROR] HTTPConnectionPool(host='127.0.0.1', port=37013): Max retries exceeded with url: /semantic_completion_available (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0e940b82e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2021-02-15 10:14:39,953 [Thread-28] [DEBUG] Starting new HTTP connection (54): 127.0.0.1:37013
2021-02-15 10:14:39,954 [MainThread] [ERROR] HTTPConnectionPool(host='127.0.0.1', port=37013): Max retries exceeded with url: /semantic_completion_available (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0e940dbfa0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2021-02-15 10:14:39,979 [Thread-15] [DEBUG] Starting new HTTP connection (55): 127.0.0.1:37013
2021-02-15 10:14:40,005 [MainThread] [ERROR] HTTPConnectionPool(host='127.0.0.1', port=37013): Max retries exceeded with url: /semantic_completion_available (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0e940b8df0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2021-02-15 10:14:40,007 [Thread-29] [DEBUG] Starting new HTTP connection (56): 127.0.0.1:37013
2021-02-15 10:14:40,009 [Thread-9] [DEBUG] Starting new HTTP connection (57): 127.0.0.1:37013
2021-02-15 10:14:40,009 [MainThread] [ERROR] HTTPConnectionPool(host='127.0.0.1', port=37013): Max retries exceeded with url: /semantic_completion_available (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0e940b85b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2021-02-15 10:14:40,036 [Thread-30] [DEBUG] Starting new HTTP connection (58): 127.0.0.1:37013
2021-02-15 10:14:41,382 [MainThread] [ERROR] HTTPConnectionPool(host='127.0.0.1', port=37013): Max retries exceeded with url: /semantic_completion_available (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0e940db7c0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2021-02-15 10:14:41,484 [Thread-16] [DEBUG] Starting new HTTP connection (59): 127.0.0.1:37013
2021-02-15 10:14:41,518 [MainThread] [ERROR] HTTPConnectionPool(host='127.0.0.1', port=37013): Max retries exceeded with url: /completions (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0e940db220>: Failed to establish a new connection: [Errno 111] Connection refused'))
2021-02-15 10:14:42,165 [Thread-31] [DEBUG] Starting new HTTP connection (60): 127.0.0.1:37013
2021-02-15 10:14:56,514 [Thread-2] [DEBUG] Starting new HTTP connection (61): 127.0.0.1:37013
2021-02-15 10:14:56,532 [Thread-17] [DEBUG] Starting new HTTP connection (62): 127.0.0.1:37013
2021-02-15 10:15:03,460 [Thread-3] [DEBUG] Starting new HTTP connection (63): 127.0.0.1:37013
2021-02-15 10:15:03,477 [Thread-18] [DEBUG] Starting new HTTP connection (64): 127.0.0.1:37013
2021-02-15 10:15:03,479 [Thread-10] [DEBUG] Starting new HTTP connection (65): 127.0.0.1:37013
2021-02-15 10:15:03,480 [Thread-19] [DEBUG] Starting new HTTP connection (66): 127.0.0.1:37013
2021-02-15 10:15:03,480 [MainThread] [ERROR] HTTPConnectionPool(host='127.0.0.1', port=37013): Max retries exceeded with url: /shutdown (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0e741329d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2021-02-15 10:15:30,921 [Thread-6] [DEBUG] Starting new HTTP connection (5): 127.0.0.1:39737
2021-02-15 10:15:30,925 [MainThread] [ERROR] HTTPConnectionPool(host='127.0.0.1', port=39737): Max retries exceeded with url: /semantic_completion_available (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3ca8c80940>: Failed to establish a new connection: [Errno 111] Connection refused'))
2021-02-15 10:15:30,947 [Thread-7] [DEBUG] Starting new HTTP connection (6): 127.0.0.1:39737
2021-02-15 10:15:36,030 [MainThread] [ERROR] HTTPConnectionPool(host='127.0.0.1', port=39737): Max retries exceeded with url: /semantic_completion_available (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3ca8c0c190>: Failed to establish a new connection: [Errno 111] Connection refused'))
2021-02-15 10:15:36,162 [Thread-8] [DEBUG] Starting new HTTP connection (7): 127.0.0.1:39737
2021-02-15 10:15:36,263 [MainThread] [ERROR] HTTPConnectionPool(host='127.0.0.1', port=39737): Max retries exceeded with url: /semantic_completion_available (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3ca8c0ce50>: Failed to establish a new connection: [Errno 111] Connection refused'))
2021-02-15 10:15:36,558 [Thread-5] [DEBUG] Starting new HTTP connection (8): 127.0.0.1:39737
2021-02-15 10:15:50,004 [Thread-2] [DEBUG] Starting new HTTP connection (9): 127.0.0.1:39737
2021-02-15 10:15:50,021 [Thread-11] [DEBUG] Starting new HTTP connection (10): 127.0.0.1:39737
2021-02-15 10:15:50,024 [Thread-12] [DEBUG] Starting new HTTP connection (11): 127.0.0.1:39737
2021-02-15 10:15:50,026 [Thread-13] [DEBUG] Starting new HTTP connection (12): 127.0.0.1:39737
2021-02-15 10:15:50,027 [MainThread] [ERROR] HTTPConnectionPool(host='127.0.0.1', port=39737): Max retries exceeded with url: /shutdown (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3ca83c69d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2021-02-15 15:36:56,855 [MainThread] [DEBUG] False
2021-02-15 15:36:56,901 [MainThread] [INFO] Configuration file has been loaded...
2021-02-15 15:36:56,904 [MainThread] [INFO] Script Runner has started
2021-02-15 15:36:57,093 [MainThread] [INFO] Assigning functions
2021-02-15 15:37:11,180 [MainThread] [DEBUG] Elapsed time: 2.6723 seconds
2021-02-15 15:37:13,509 [MainThread] [DEBUG] Elapsed time: 0.0000 seconds
2021-02-15 15:37:19,729 [MainThread] [DEBUG] Elapsed time: 0.0000 seconds
2021-02-15 15:44:03,622 [MainThread] [DEBUG] False
2021-02-15 15:44:03,642 [MainThread] [INFO] Configuration file has been loaded...
2021-02-15 15:44:03,643 [MainThread] [INFO] Script Runner has started
2021-02-15 15:44:03,709 [MainThread] [INFO] Assigning functions
2021-02-15 15:44:16,355 [MainThread] [INFO] Output: ['Get-CimInstance : Access denied ', 'At C:\\WINDOWS\\system32\\WindowsPowerShell\\v1.0\\Modules\\BitLocker\\BitLocker.psm1:144 char:13', '+             Get-CimInstance `', '+             ~~~~~~~~~~~~~~~~~', '    + CategoryInfo          : PermissionDenied: (root\\cimv2\\Secu...cryptableVolume:String) [Get-CimInstance], CimException', '    + FullyQualifiedErrorId : HRESULT 0x80041003,Microsoft.Management.Infrastructure.CimCmdlets.GetCimInstanceCommand', ' ', 'Get-Win32EncryptableVolumeInternal :  does not have an associated BitLocker volume.', 'At C:\\WINDOWS\\system32\\WindowsPowerShell\\v1.0\\Modules\\BitLocker\\BitLocker.psm1:696 char:42', '+ ...       $AllWin32EncryptableVolume = Get-Win32EncryptableVolumeInternal', '+                                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~', '    + CategoryInfo          : NotSpecified: (:) [Write-Error], COMException', '    + FullyQualifiedErrorId : System.Runtime.InteropServices.COMException,Get-Win32EncryptableVolumeInternal', '']
2021-02-15 15:44:16,356 [MainThread] [INFO] [':', 'not', 'C:\\WINDOWS\\system32\\WindowsPowerShell\\v1.0\\Modules\\BitLocker\\BitLocker.psm1:696']
2021-02-15 15:53:22,473 [MainThread] [DEBUG] False
2021-02-15 15:53:22,493 [MainThread] [INFO] Configuration file has been loaded...
2021-02-15 15:53:22,494 [MainThread] [INFO] Script Runner has started
2021-02-15 15:53:22,562 [MainThread] [INFO] Assigning functions
2021-02-15 15:53:34,282 [MainThread] [DEBUG] Elapsed time: 3.1805 seconds
2021-02-15 15:55:42,164 [MainThread] [DEBUG] Elapsed time: 0.0000 seconds
2021-02-15 15:56:24,196 [MainThread] [DEBUG] True
2021-02-15 15:56:24,308 [MainThread] [INFO] Configuration file has been loaded...
2021-02-15 15:56:24,311 [MainThread] [INFO] Script Runner has started
2021-02-15 15:56:24,433 [MainThread] [INFO] Assigning functions
2021-02-15 15:58:27,083 [MainThread] [DEBUG] True
2021-02-15 15:58:27,127 [MainThread] [INFO] Configuration file has been loaded...
2021-02-15 15:58:27,128 [MainThread] [INFO] Script Runner has started
2021-02-15 15:58:27,179 [MainThread] [INFO] Assigning functions
2021-02-15 15:59:21,166 [MainThread] [DEBUG] True
2021-02-15 15:59:21,219 [MainThread] [INFO] Configuration file has been loaded...
2021-02-15 15:59:21,219 [MainThread] [INFO] Script Runner has started
2021-02-15 15:59:21,266 [MainThread] [INFO] Assigning functions
2021-02-15 15:59:50,454 [MainThread] [DEBUG] True
2021-02-15 15:59:50,492 [MainThread] [INFO] Configuration file has been loaded...
2021-02-15 15:59:50,492 [MainThread] [INFO] Script Runner has started
2021-02-15 15:59:50,546 [MainThread] [INFO] Assigning functions
2021-02-15 16:12:25,257 [MainThread] [DEBUG] False
2021-02-15 16:12:25,278 [MainThread] [INFO] Configuration file has been loaded...
2021-02-15 16:12:25,279 [MainThread] [INFO] Script Runner has started
2021-02-15 16:12:25,345 [MainThread] [INFO] Assigning functions
2021-02-15 16:12:30,005 [Finding Names] [INFO] ['\n', 'NT AUTHORITY\\Authenticated Users', 'NT AUTHORITY\\INTERACTIVE', 'postgres', 'Administrator', 'Michael   (Current User)']
2021-02-15 16:12:30,005 [MainThread] [DEBUG] Elapsed time: 0.5473 seconds
2021-02-15 16:12:36,248 [Finding Groups] [INFO] ['__vmware__', 'Administrators', 'Distributed COM Users', 'Event Log Readers', 'Guests', 'Hyper-V Administrators', 'IIS_IUSRS', 'Performance Log Users', 'Performance Monitor Users', 'Remote Management Users', 'Ssh Users', 'System Managed Accounts Group', 'Users']
2021-02-15 16:12:36,248 [Finding Groups] [INFO] {}
2021-02-15 16:12:40,896 [MainThread] [DEBUG] Elapsed time: 0.0000 seconds
2021-02-15 16:31:03,691 [MainThread] [DEBUG] False
2021-02-15 16:31:03,712 [MainThread] [INFO] Configuration file has been loaded...
2021-02-15 16:31:03,712 [MainThread] [INFO] Script Runner has started
2021-02-15 16:31:03,780 [MainThread] [INFO] Assigning functions
2021-02-15 16:31:11,366 [MainThread] [DEBUG] Elapsed time: 0.0000 seconds
2021-02-18 12:17:27,907 [MainThread] [DEBUG] True
2021-02-18 12:17:27,986 [MainThread] [INFO] Configuration file has been loaded...
2021-02-18 12:17:27,989 [MainThread] [INFO] Script Runner has started
2021-02-18 12:17:28,101 [MainThread] [INFO] Assigning functions
                                                                                                                                                     self.actionHow_To_Use_Program.triggered.connect(lambda: showHOWTO())
        self.actionAbout_Creator.triggered.conne2021-02-24 12:01:36,599 [MainThread] [DEBUG] False
2021-02-24 12:01:36,688 [MainThread] [INFO] Configuration file has been loaded...
2021-02-24 12:01:36,689 [MainThread] [INFO] Script Runner has started
2021-02-24 12:01:36,733 [MainThread] [INFO] Assigning functions
2021-02-24 12:01:46,234 [MainThread] [DEBUG] True
2021-02-24 12:01:46,330 [MainThread] [INFO] Configuration file has been loaded...
2021-02-24 12:01:46,331 [MainThread] [INFO] Script Runner has started
2021-02-24 12:01:46,375 [MainThread] [INFO] Assigning functions
2021-02-24 12:01:55,862 [MainThread] [DEBUG] Elapsed time: 0.0043 seconds
2021-02-24 12:12:09,582 [MainThread] [DEBUG] True
2021-02-24 12:12:09,674 [MainThread] [INFO] Configuration file has been loaded...
2021-02-24 12:12:09,674 [MainThread] [INFO] Script Runner has started
2021-02-24 12:12:09,718 [MainThread] [INFO] Assigning functions
2021-02-26 13:10:42,247 [MainThread] [DEBUG] False
2021-02-26 13:10:42,362 [MainThread] [INFO] Configuration file has been loaded...
2021-02-26 13:10:42,365 [MainThread] [INFO] Script Runner has started
2021-02-26 13:10:42,426 [MainThread] [INFO] Assigning functions
2021-02-26 20:20:50,699 [MainThread] [DEBUG] True
2021-02-26 20:20:50,784 [MainThread] [INFO] Configuration file has been loaded...
2021-02-26 20:20:50,787 [MainThread] [INFO] Script Runner has started
2021-02-26 20:20:50,901 [MainThread] [INFO] Assigning functions
